tests/api/test_api.py

```diff
@@ -96,11 +96,11 @@ async def test_if_get_fsc_revocation_job_returns_success_status_with_result():
         mock_service = MagicMock(get_request_by_request_id=MagicMock(return_value=erica_request))
         get_service_mock.return_value = FreischaltCodeService(service=mock_service)
         response = await get_fsc_revocation_job(request_id)
-        assert response.processStatus == JobState.SUCCESS
-        assert response.result.idnr == tax_id_number
+        assert response.process_status == JobState.SUCCESS
+        assert response.result.tax_id_number == tax_id_number
```

Comment:
So the api will not return camelCase? Or is that even one more level up? (And which would be that level? 🤔 )

Reply:
No, the Api will return camelCase because the response inherit from BaseDto -> CamelCaseInput

Reply:
Shouldn't this be `assert response.result.taxIdNumber == tax_id_number` then? (Because `response` is the response of the api?

Reply:
As I understood the CamelCaseInput it transforms camelCase to snake_case attributes when constructing the object and it transforms the snake_case attributes to camelCase when being parsed as JSON.

Here we test the API functions but they still return the normal object (snake_case). In the contract tests we still expect camelCase

Reply:
Ah right - on Json parsing. Can we maybe add a test that makes that clear? I honestly wouldn't know what to test - maybe the CamelCaseInput directly?

Reply:
Maybe manually doing a JSON serialization of the returned object?

Reply:
Yes, let's test the CamelCaseInput directly and do the JSON serialization there?